hedera-mirror-rosetta/app/services/construction/token_burn_mint_transaction_constructor_test.go

```diff
@@ -487,17 +488,27 @@ func assertTokenBurnMintTransaction(
 
 	var payer string
 	var token string
+	var value string
 
 	switch tx := actual.(type) {
 	case *hedera.TokenBurnTransaction:
 		payer = tx.GetTransactionID().AccountID.String()
 		token = tx.GetTokenID().String()
+		value = fmt.Sprintf("-%d", tx.GetAmount())
```

Comment:
nit: shared logic repeated a few times.
Could pull out `fmt.Sprintf("-%d", ...)` into a format method

Reply:
don't feel we should wrap it as a single line function. perhaps I should rewrite it as so it doesn't look unnatural?

 ```suggestion
		value = fmt.Sprintf("%d", -tx.GetAmount())
```

Reply:
Non blocking so as is works for me